index.html

```diff
@@ -245,6 +245,28 @@ <h4>Andrew Poelstra</h4>
             </div>
           </div>
 
+          <div class="col-sm-4">
+            <div class="team-member">
+              <img class="mx-auto rounded-circle" src="img/team/bosworth.jpg" alt="profile photo">
+              <h4>Alex Bosworth</h4>
+              <ul class="list-inline social-buttons">
+                <li class="list-inline-item">
+                  <a target="_blank" href="https://github.com/alexbosworth">
+                    <i class="fa fa-github"></i>
+                  </a>
+                </li>
+                <li class="list-inline-item">
+                  <a target="_blank" href="https://twitter.com/alexbosworth">
+                    <i class="fa fa-twitter"></i>
+                  </a>
+                </li>
+              </ul>
+              <div class="bio">
+                Alex is a lightning infrastructure lead at Lightning Labs. Previously, he created yalls.org, submarineswaps.org, and helped develop a blockchain project used to provide transparency for the Royal Mint of the U.K.
```

Comment:
s/a lightning infrastructure lead/the lightning infrastructure lead/

Reply:
Also, I'm not sure if working on the Royal Mint project is what he'd choose to put in his bio. I think I'd just leave it at "Previously, he created yalls.org, htlc.me and submarineswaps.org"

Reply:
https://lightning.engineering/team.html